### ISSUE TITLE
大佬 我的配置也是跟你的一样安装卡在白苹果logo

### ISSUE THREAD

**Issue by tang-zack** [liangzhenduo/hackintosh-efi#12]
配置跟你一样    安装的是最新的Install_macOS-Big-Sur--11.5.2(20G95) 结果直接卡在白苹果logo这里 键盘和鼠标都失效了

**Comment by liangzhenduo0608** [comment]
我升级到了11.5.2没发现问题
是安装系统阶段卡住了吗？试试加入-v看一下详细信息
参考 https://blog.daliansky.net/Common-problems-and-solutions-in-macOS-BigSur-11.0-installation.html

**Comment by tang-zack** [comment]
是的 大佬 我们配置一模一样的  我这边也加了-v  好像是卡在+++的位置 我晚上回去在测试一下 然后截图给您  帮我看一下

**Comment by tang-zack** [comment]
![image](https://user-images.githubusercontent.com/76250616/149149802-ccb95b81-28ce-4cdf-b7f1-34c8855ab831.png)

**Comment by liangzhenduo0608** [comment]
> ![image](https://user-images.githubusercontent.com/76250616/149149802-ccb95b81-28ce-4cdf-b7f1-34c8855ab831.png)

卡在 [EB|#LOG:EXITBS:START]，上面那个链接里给了解决方案，应该是BIOS没禁用CFG-Lock

**Comment by tang-zack** [comment]
我在bios里面的CFG Lock 是关闭了的 而且 config里面也设置了的
